dockerfile/Dockerfile

```diff
@@ -159,3 +159,4 @@ ENV PATH $PATH:$GOPATH/bin
 # Install Go's dependency management tool.
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
+COPY dockerfile/script.sh /cmd/script.sh
```

Comment:
Is it more precise to rename script.sh into build_and_test.sh, and actually we can use it for our daily development work in addition to CI.